recipes/find_circ/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "find_circ" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/marvin-jens/find_circ.git
+  git_rev: 8655dca54970fcf7e92e22fbf57e1188724dda7d
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  host:
+    - python ==2.7.15
+    - pip
+  run:
+    - python ==2.7.15
+    - numpy ==1.16.4
+    - pysam ==0.15.2
+
+test:
+  commands:
+    - python --version
```

Comment:
The test needs to be something specific to the package to check it installed correctly, even if it's just the imports like [this](https://github.com/bioconda/bioconda-recipes/blob/57e2c903407676cca776299a9e0bf82036c0631a/recipes/scanpy-scripts/meta.yaml#L73). 

The Python version check won't work for that.